frontend/src/components/Graph.tsx

```diff
@@ -1,14 +1,29 @@
 import React, { useEffect, useState } from 'react'
 import axios from 'axios'
-import Graphin, { Behaviors } from '@antv/graphin'
+import Graphin, { Behaviors, GraphinContext } from '@antv/graphin'
+import { MiniMap, Tooltip } from '@antv/graphin-components'
+import Card from './Card'
 import dataParser from '../helpers/dataParser'
 import G6Data from '../interfaces/G6Data'
 
-const { ZoomCanvas } = Behaviors
+const { ZoomCanvas, ActivateRelations, Hoverable, ClickSelect } = Behaviors
 
 const link =
   'https://gist.githubusercontent.com/IdreesSamadi/6aa2e5f0f8c3828b41f1e3446d2002cd/raw/messages.json'
 
+const CustomTooltip = () => {
+  const { tooltip } = React.useContext(GraphinContext)
+  const context = tooltip.node
+  const { item } = context
+  const model = item && item.getModel()
+
+  return (
+    <div>
+      <Card id={model.id} />
+    </div>
+  )
+}
+
```

Comment:
its better to keep each file slim, and to have one component per file. move the tooltip to its own file and import in graph.tsx.

Reply:
maybe also move the minimap to the right. we have the navigation to the left seem like too many stuff

Reply:
I vote for bottom right too.

Reply:
![image](https://user-images.githubusercontent.com/57683502/135311202-71dc9a54-8439-4e6e-ac62-f367994e8c24.png)

Reply:
nice